packages/components/src/popover/types.ts

```diff
@@ -31,20 +35,11 @@ export type PopoverProps = {
 	 */
 	__unstableSlotName?: string;
 	/**
-	 * An object extending a `DOMRect` with an additional optional `ownerDocument`
-	 * property, used to specify a fixed popover position.
+	 * The element that should be used by the popover as its anchor. It can either
+	 * be an `Element` or, alternatively, a `VirtualElement` — ie. an object with
+	 * the `getBoundingClientRect()` and the `ownerDocument` properties defined.
```

Comment:
Should we add a note about the ref/state thing in this prop description as well?

Reply:
Absolutely. Done in https://github.com/WordPress/gutenberg/pull/43691/commits/0dc076a2f4ab5b2b69a74b6c777340bf6571f491